src/egl/drivers/dri2/platform_android_gralloc.cpp

```diff
@@ -0,0 +1,23 @@
+#include "egl_dri2.h"
+#include <cutils/log.h>
+extern "C" int resolve_gralloc1_functions(struct dri2_egl_display *dri2_dpy);
+
+int resolve_gralloc1_functions(struct dri2_egl_display *dri2_dpy){
```

Comment:
BTW What was the reason to make this one a separate file? It's just one function so maybe better to include it in platform_android.cpp?